docs/input.rst

```diff
@@ -150,15 +144,20 @@ You can obtain information about the existing matched publications with
    for pub_info in matched_pubs:
       pub_name = pub_info['name']
 
-Class reference: Input, SampleIterator, ValidSampleIterator
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Class reference: Input, Samples, SampleIterator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Input class
 ^^^^^^^^^^^
 
 .. autoclass:: rticonnextdds_connector.Input
    :members:
 
+Samples class
+^^^^^^^^^^^^^
+
+.. autoclass:: rticonnextdds_connector.Samples
+   :members:
 
 SampleIterator class
 ^^^^^^^^^^^^^^^^^^^^
```

Comment:
Having SampleIterator in the docs might be confusing unless we include a sentence about how it is used (i.e., used by the __iter__ method on input._samples)

Reply:
"	A SampleIterator provides access to the data received by an input.
	SampleIterator is the iterator type of :attr:`Input.samples`."